docker-compose.yml

```diff
@@ -7,6 +7,7 @@ services:
       - "5001:80"
     environment:
       ASPNETCORE_ENVIRONMENT: "Development"
+      ASPNETCORE_HTTP_PORTS: "80"
```

Comment:
Assuming this all works fine running locally?

Reply:
I tested this change yesterday after you suggested trying it locally, and it appears to work fine, including PDF generation.